src/annotations.cr

```diff
@@ -45,6 +45,13 @@ end
 #
 # `@[Link(framework: "Cocoa")]` will pass `-framework Cocoa` to the linker.
 #
+# `@[Link(dll: "gc.dll")]` will copy `gc.dll` to any built program. The DLL name
+# must use `.dll` as its file extension and cannot contain any directory
+# separators. The actual DLL is searched among `CRYSTAL_LIBRARY_PATH`, the
+# compiler itself's directory, and `PATH` in that order; a warning is printed if
```

Comment:
**polish(language):** "the compiler itself's directory" doesn't sound like a proper phrase in English to me.

```suggestion
# compiler's own directory, and `PATH` in that order; a warning is printed if
```